charts/cluster-registrator/templates/job.yaml

```diff
@@ -6,13 +6,6 @@ metadata:
   name: {{ .Values.namespace }}
 {{- end }}
 ---
-{{- if not (lookup "v1" "Namespace" "" "nirmata") }}
```

Comment:
I feel we still should have an option to create nirmata nsp if it does not exist, maybe another helm param "createNirmataNsp"

Reply:
the requirement was that they dont want the chart to create the chart on its own, it wasnt making much sense to allow for user to have `createNs` (bool: default to true) and `namespace` (string: default to nirmata). With both the options if the user says createNs:true namespace:test, the chart would create Nirmata namespace still. If we add another option like createNirmataNsp, it would do the same thing but just for nirmata ns explicitly. WDYT?

Reply:
Yes, maybe we ignore this then.